code/modules/mob/observer/freelook/marker/signal.dm

```diff
@@ -260,6 +262,21 @@ GLOBAL_LIST_INIT(signal_sprites, list("markersignal-1",
 
 	to_chat(src, SPAN_DANGER("Error: No marker found!"))
 
+/mob/observer/eye/signal/verb/jump_to_alive_necro()
+	set name = "Jump to Necromorph"
```

Comment:
change this to use major vessels

may also possibly want to only jump to things that need a controller